src/status_im/integration_test/wallet_test.cljs

```diff
@@ -12,40 +11,21 @@
     status-im.subs.root
     [test-helpers.integration :as h]))
 
-;; Workaround to skip test. Switch to `deftest` when test is fixed.
-(defn create-wallet-account-test
-  []
+(defn assert-accounts-contract
+  [result]
+  (is (true? (some :wallet result)))
+  (is (true? (some :chat result)))
+  (is (= 2 (count result))))
+
+(deftest check-wallet-accounts-contract
   (h/log-headline :create-wallet-account-test)
   (rf-test/run-test-async
    (h/with-app-initialized
     (h/with-account
-     (h/create-new-account!)
+     (rf/dispatch [:integration-test/dispatch-rpc
+                   {:rpc-endpoint "accounts_getAccounts"
```

Comment:
this is then all we need to write a _Contract Test_,
rpc endpoint, params (optional) and an assertion callback